aria-label for hamburger menù

massimoghm opened the pull request:
aria-label for hamburger menù

Closes https://github.com/bootscore/bootscore/issues/934

crftwrk commented:
Thank you, this is a good idea! 

If we do that, we should add the `aria-label`to all buttons in the header. This are the `search-toggler`, `account-toggler`, `mini-cart-toggler` and the `back-to-cart-toggler` located in `header-actions.php` and `header-actions-woocommerce.php` in folder `template-parts/header/`.

Want to add them?

massimoghm commented:
Updated, check if new escape method is correct. I think is more codex style.
Updated version to 6.1.0.

If you remember other accessibility points, tell me :)

Best regards

crftwrk commented:
Next release will be on Feb. 24 alongside the WooCommerce 9.7 update. Not sure if we are ready to ship v6.1 until there. So version 6.0.5 is more safe.

massimoghm commented:
Revert to 6.0.5 release :)

crftwrk commented on header.php:
```diff
@@ -87,7 +87,7 @@
 
           <!-- Navbar Toggler -->
-          <button class="<?= apply_filters('bootscore/class/header/button', 'btn btn-outline-secondary', 'nav-toggler'); ?> <?= apply_filters('bootscore/class/header/navbar/toggler/breakpoint', 'd-lg-none'); ?> ms-1 ms-md-2 nav-toggler" type="button" data-bs-toggle="offcanvas" data-bs-target="#offcanvas-navbar" aria-controls="offcanvas-navbar">
+          <button class="<?= apply_filters('bootscore/class/header/button', 'btn btn-outline-secondary', 'nav-toggler'); ?> <?= apply_filters('bootscore/class/header/navbar/toggler/breakpoint', 'd-lg-none'); ?> ms-1 ms-md-2 nav-toggler" type="button" data-bs-toggle="offcanvas" data-bs-target="#offcanvas-navbar" aria-controls="offcanvas-navbar" aria-label="<?php esc_html_e( 'Toggle main menu', 'bo ...
```
Why using `esc_html_e`here while others uses `esc_attr_e`?

massimoghm replied:
Changed

crftwrk reviewed (approved):
Thank you 🙏. Just fixed missing close `"` in [header.php](https://github.com/bootscore/bootscore/pull/936/commits/7ef8b91ca37244e1b257061db7116833f47dab84)
